CONTRIBUTING.md

```diff
@@ -17,21 +17,16 @@ issue trackers, chatrooms, and mailing lists.
  - Try to update the .gitignore if you're adding different formats of content.
  - Create a PR :bowtie:
 
-## Contributing Co
+## Contributing Guidelines
 
-
-Make your change, with new passing tests. Follow the [style guide][style].
+Make your change and Follow the [style guide][style].
 
   [style]: https://github.com/thoughtbot/guides/tree/master/style
 
-Mention how your changes affect the project to other developers and users in the
+Mention how your changes affect the project repository impacting other developers and users in the
```

Comment:
This is looking good. I left a few comments that should be addressed before this PR gets merged. 😸